### PR TITLE
chore: add volunteer sgkandale

### PR DESCRIPTION
**Pull request by sgkandale**

### CODE REVIEW

File: data/users/sgkandale.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "bio": "Backend Engineer",
+  "image": "https://avatars.githubusercontent.com/u/40430224",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["Golang", "Javascript"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["Backend", "Frontend"]
```
Comment (codewithvoid) [inline data/users/sgkandale.json:12]: You can remove this..as this is already mentioned in the title

File: data/users/sgkandale.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "slug": "web-development",
+      "topics": ["Javascript", "ReactJS"]
```
Comment (codewithvoid) [inline data/users/sgkandale.json:16]: Remove javascript as it's already mentioned in programming

File: data/users/sgkandale.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Shantanu Kandale",
+  "bio": "Backend Engineer",
+  "image": "https://avatars.githubusercontent.com/u/40430224",
```
Comment (codewithvoid) [inline data/users/sgkandale.json:4]: use this github image url instead https://avatars.githubusercontent.com/u/40430224?v=4

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by codewithvoid** [comment]
Thanks for volunteering @sgkandale . Added some comments to the file. PTAL.

**Comment by sgkandale** [comment]
Thanks for the feedback @codewithvoid , I've made the necessary changes.

**Comment by codewithvoid** [comment]
Perfect. Welcome to community!

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**